src/App.js

```diff
@@ -16,53 +16,48 @@ function App() {
   const [background, setBackground] = useState("");
   const [isLoading, setIsLoading] = useState(false);
 
-  const searchPressed = () => {
+  const searchHandler = async () => {
     setIsLoading(true);
-    fetch(`${api.base}weather?q=${search}&units=metric&APPID=${api.key}`)
-      .then((res) => res.json())
+    try {
+      await fetch(
```

Comment:
"async await" can go with ".then" in theory but in praxis it's never the case, it's hard to maintain and follow such code. So try only using async await please